[
    ">> 343    src/mpi4py.MPI.c:163192:33: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     344    163192 |   __pyx_type_6mpi4py_3MPI_Pickle.tp_print = 0;\n     345           |                                 ^\n     346    In file included from src/MPI.c:4:\n     347    src/mpi4py.MPI.c: In function '__Pyx_ParseOptionalKeywords':\n348    src/mpi4py.MPI.c:175146:21: warning: '_PyUnicode_get_wstr_length' i\n            s deprecated [-Wdeprecated-declarations]\n     349    175146 |                     (PyUnicode_GET_SIZE(**name) != PyUnico\n            de_GET_SIZE(key)) ? 1 :\n\n     ...",
    "configtest.o\n     39     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc _configtest.o -llmpe -o _confi\n            gtest\n  >> 40     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-re\ndhat-linux/9/ld: cannot find -llmpe\n  >> 41     collect2: error: ld returned 1 exit status\n     42     failure.\n     43     removing: _configtest.c _configtest.o\n     44     building 'mpe' dylib library\n     45     creating build/temp.linux-x86_64-3.9\n     46     creating build/temp.linux-x86_64-3.9/src",
    "39     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc _configtest.o -llmpe -o _confi\n            gtest\n  >> 40     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-re\n            dhat-linux/9/ld: cannot find -llmpe\n>> 41     collect2: error: ld returned 1 exit status\n     42     failure.\n     43     removing: _configtest.c _configtest.o\n     44     building 'mpe' dylib library\n     45     creating build/temp.linux-x86_64-3.9\n     46     creating build/temp.linux-x86_64-3.9/src\n     47     creating build/temp.linux-x86_64-3.9/src/lib-pmpi",
    "configtest.o\n     53     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc _configtest.o -lvt-mpi -o _con\n            figtest\n  >> 54     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-re\ndhat-linux/9/ld: cannot find -lvt-mpi\n  >> 55     collect2: error: ld returned 1 exit status\n     56     failure.\n     57     removing: _configtest.c _configtest.o\n     58     checking for library 'vt.mpi' ...\n     59     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw",
    "53     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc _configtest.o -lvt-mpi -o _con\n            figtest\n  >> 54     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-re\n            dhat-linux/9/ld: cannot find -lvt-mpi\n>> 55     collect2: error: ld returned 1 exit status\n     56     failure.\n     57     removing: _configtest.c _configtest.o\n     58     checking for library 'vt.mpi' ...\n     59     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc -Wno-unused-result -Wsign-comp",
    "configtest.o\n     60     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc _configtest.o -lvt.mpi -o _con\n            figtest\n  >> 61     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-re\ndhat-linux/9/ld: cannot find -lvt.mpi\n  >> 62     collect2: error: ld returned 1 exit status\n     63     failure.\n     64     removing: _configtest.c _configtest.o\n     65     building 'vt' dylib library\n     66     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw",
    "60     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc _configtest.o -lvt.mpi -o _con\n            figtest\n  >> 61     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-re\n            dhat-linux/9/ld: cannot find -lvt.mpi\n>> 62     collect2: error: ld returned 1 exit status\n     63     failure.\n     64     removing: _configtest.c _configtest.o\n     65     building 'vt' dylib library\n     66     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc -Wno-unused-result -Wsign-comp",
    "configtest.o\n     70     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc _configtest.o -lvt-mpi -o _con\n            figtest\n  >> 71     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-re\ndhat-linux/9/ld: cannot find -lvt-mpi\n  >> 72     collect2: error: ld returned 1 exit status\n     73     failure.\n     74     removing: _configtest.c _configtest.o\n     75     checking for library 'vt.mpi' ...\n     76     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw",
    "70     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc _configtest.o -lvt-mpi -o _con\n            figtest\n  >> 71     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-re\n            dhat-linux/9/ld: cannot find -lvt-mpi\n>> 72     collect2: error: ld returned 1 exit status\n     73     failure.\n     74     removing: _configtest.c _configtest.o\n     75     checking for library 'vt.mpi' ...\n     76     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc -Wno-unused-result -Wsign-comp",
    "configtest.o\n     77     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc _configtest.o -lvt.mpi -o _con\n            figtest\n  >> 78     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-re\ndhat-linux/9/ld: cannot find -lvt.mpi\n  >> 79     collect2: error: ld returned 1 exit status\n     80     failure.\n     81     removing: _configtest.c _configtest.o\n     82     building 'vt-mpi' dylib library\n     83     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw",
    "77     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc _configtest.o -lvt.mpi -o _con\n            figtest\n  >> 78     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-re\n            dhat-linux/9/ld: cannot find -lvt.mpi\n>> 79     collect2: error: ld returned 1 exit status\n     80     failure.\n     81     removing: _configtest.c _configtest.o\n     82     building 'vt-mpi' dylib library\n     83     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc -Wno-unused-result -Wsign-comp",
    "configtest.o\n     87     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc _configtest.o -lvt-hyb -o _con\n            figtest\n  >> 88     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-re\ndhat-linux/9/ld: cannot find -lvt-hyb\n  >> 89     collect2: error: ld returned 1 exit status\n     90     failure.\n     91     removing: _configtest.c _configtest.o\n     92     checking for library 'vt.ompi' ...\n     93     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw",
    "87     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc _configtest.o -lvt-hyb -o _con\n            figtest\n  >> 88     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-re\n            dhat-linux/9/ld: cannot find -lvt-hyb\n>> 89     collect2: error: ld returned 1 exit status\n     90     failure.\n     91     removing: _configtest.c _configtest.o\n     92     checking for library 'vt.ompi' ...\n     93     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc -Wno-unused-result -Wsign-comp",
    "configtest.o\n     94     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc _configtest.o -lvt.ompi -o _co\n            nfigtest\n  >> 95     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-re\ndhat-linux/9/ld: cannot find -lvt.ompi\n  >> 96     collect2: error: ld returned 1 exit status\n     97     failure.\n     98     removing: _configtest.c _configtest.o\n     99     building 'vt-hyb' dylib library\n     100    /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw",
    "94     /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc _configtest.o -lvt.ompi -o _co\n            nfigtest\n  >> 95     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-re\n            dhat-linux/9/ld: cannot find -lvt.ompi\n>> 96     collect2: error: ld returned 1 exit status\n     97     failure.\n     98     removing: _configtest.c _configtest.o\n     99     building 'vt-hyb' dylib library\n     100    /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc -Wno-unused-result -Wsign-comp",
    "ags,-R/p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-lib\n            ensemble/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/python-3.9.9\n            -3o76m4utiyrsvuxwx2ytcv7sd3g35si6/lib -o _configtest\n     157    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-re\n            dhat-linux/9/ld: _configtest.o: in function `main':\n  >> 158    /tmp/koparasy/spack-stage/spack-stage-py-mpi4py-3.0.0-oqkbkm6n2lf25\nsukpm6bu2p23jq7tszk/spack-src/_configtest.c:5: undefined reference\n            to `MPI_Status_c2f'\n  >> 159    collect2: error: ld returned 1 exit status\n     160    failure.\n     161    removing: _configtest.c _configtest.o\n     162    checking for function 'MPI_Status_f2c' ...\n     163    /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb",
    "-3o76m4utiyrsvuxwx2ytcv7sd3g35si6/lib -o _configtest\n     157    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-re\n            dhat-linux/9/ld: _configtest.o: in function `main':\n  >> 158    /tmp/koparasy/spack-stage/spack-stage-py-mpi4py-3.0.0-oqkbkm6n2lf25\n            sukpm6bu2p23jq7tszk/spack-src/_configtest.c:5: undefined reference\n            to `MPI_Status_c2f'\n>> 159    collect2: error: ld returned 1 exit status\n     160    failure.\n     161    removing: _configtest.c _configtest.o\n     162    checking for function 'MPI_Status_f2c' ...\n     163    /p/vast1/koparasy/BUILD/spack_cluster_installer/quartz/py-libensemb\n            le/spack_env_dir/linux-rhel7-broadwell/gcc-9.3.1/mpich-3.1.1-qao2iw\n            6bdjj5a6bwmda4f7xqf6lhvu2z/bin/mpicc -Wno-unused-result -Wsign-comp",
    "te: declared here\n     249      334 | Py_DEPRECATED(3.0) PyAPI_FUNC(int) PyObject_CheckReadBuffer\n            (PyObject *obj);\n     250          |                                    ^~~~~~~~~~~~~~~~~~~~~~~~\n     251    In file included from src/MPI.c:4:\n     252    src/mpi4py.MPI.c: In function 'PyInit_MPI':\n>> 253    src/mpi4py.MPI.c:163012:19: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     254    163012 |   PyMPIStatus_Type.tp_print = 0;\n     255           |                   ^\n  >> 256    src/mpi4py.MPI.c:163016:21: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     257    163016 |   PyMPIDatatype_Type.tp_print = 0;",
    "251    In file included from src/MPI.c:4:\n     252    src/mpi4py.MPI.c: In function 'PyInit_MPI':\n  >> 253    src/mpi4py.MPI.c:163012:19: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     254    163012 |   PyMPIStatus_Type.tp_print = 0;\n     255           |                   ^\n>> 256    src/mpi4py.MPI.c:163016:21: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     257    163016 |   PyMPIDatatype_Type.tp_print = 0;\n     258           |                     ^\n  >> 259    src/mpi4py.MPI.c:163020:20: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     260    163020 |   PyMPIRequest_Type.tp_print = 0;",
    "254    163012 |   PyMPIStatus_Type.tp_print = 0;\n     255           |                   ^\n  >> 256    src/mpi4py.MPI.c:163016:21: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     257    163016 |   PyMPIDatatype_Type.tp_print = 0;\n     258           |                     ^\n>> 259    src/mpi4py.MPI.c:163020:20: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     260    163020 |   PyMPIRequest_Type.tp_print = 0;\n     261           |                    ^\n  >> 262    src/mpi4py.MPI.c:163025:21: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     263    163025 |   PyMPIPrequest_Type.tp_print = 0;",
    "257    163016 |   PyMPIDatatype_Type.tp_print = 0;\n     258           |                     ^\n  >> 259    src/mpi4py.MPI.c:163020:20: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     260    163020 |   PyMPIRequest_Type.tp_print = 0;\n     261           |                    ^\n>> 262    src/mpi4py.MPI.c:163025:21: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     263    163025 |   PyMPIPrequest_Type.tp_print = 0;\n     264           |                     ^\n  >> 265    src/mpi4py.MPI.c:163030:21: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     266    163030 |   PyMPIGrequest_Type.tp_print = 0;",
    "260    163020 |   PyMPIRequest_Type.tp_print = 0;\n     261           |                    ^\n  >> 262    src/mpi4py.MPI.c:163025:21: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     263    163025 |   PyMPIPrequest_Type.tp_print = 0;\n     264           |                     ^\n>> 265    src/mpi4py.MPI.c:163030:21: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     266    163030 |   PyMPIGrequest_Type.tp_print = 0;\n     267           |                     ^\n  >> 268    src/mpi4py.MPI.c:163034:20: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     269    163034 |   PyMPIMessage_Type.tp_print = 0;",
    "263    163025 |   PyMPIPrequest_Type.tp_print = 0;\n     264           |                     ^\n  >> 265    src/mpi4py.MPI.c:163030:21: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     266    163030 |   PyMPIGrequest_Type.tp_print = 0;\n     267           |                     ^\n>> 268    src/mpi4py.MPI.c:163034:20: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     269    163034 |   PyMPIMessage_Type.tp_print = 0;\n     270           |                    ^\n  >> 271    src/mpi4py.MPI.c:163038:15: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     272    163038 |   PyMPIOp_Type.tp_print = 0;",
    "266    163030 |   PyMPIGrequest_Type.tp_print = 0;\n     267           |                     ^\n  >> 268    src/mpi4py.MPI.c:163034:20: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     269    163034 |   PyMPIMessage_Type.tp_print = 0;\n     270           |                    ^\n>> 271    src/mpi4py.MPI.c:163038:15: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     272    163038 |   PyMPIOp_Type.tp_print = 0;\n     273           |               ^\n  >> 274    src/mpi4py.MPI.c:163042:18: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     275    163042 |   PyMPIGroup_Type.tp_print = 0;",
    "269    163034 |   PyMPIMessage_Type.tp_print = 0;\n     270           |                    ^\n  >> 271    src/mpi4py.MPI.c:163038:15: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     272    163038 |   PyMPIOp_Type.tp_print = 0;\n     273           |               ^\n>> 274    src/mpi4py.MPI.c:163042:18: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     275    163042 |   PyMPIGroup_Type.tp_print = 0;\n     276           |                  ^\n  >> 277    src/mpi4py.MPI.c:163046:17: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     278    163046 |   PyMPIInfo_Type.tp_print = 0;",
    "272    163038 |   PyMPIOp_Type.tp_print = 0;\n     273           |               ^\n  >> 274    src/mpi4py.MPI.c:163042:18: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     275    163042 |   PyMPIGroup_Type.tp_print = 0;\n     276           |                  ^\n>> 277    src/mpi4py.MPI.c:163046:17: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     278    163046 |   PyMPIInfo_Type.tp_print = 0;\n     279           |                 ^\n  >> 280    src/mpi4py.MPI.c:163050:23: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     281    163050 |   PyMPIErrhandler_Type.tp_print = 0;",
    "275    163042 |   PyMPIGroup_Type.tp_print = 0;\n     276           |                  ^\n  >> 277    src/mpi4py.MPI.c:163046:17: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     278    163046 |   PyMPIInfo_Type.tp_print = 0;\n     279           |                 ^\n>> 280    src/mpi4py.MPI.c:163050:23: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     281    163050 |   PyMPIErrhandler_Type.tp_print = 0;\n     282           |                       ^\n  >> 283    src/mpi4py.MPI.c:163054:17: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     284    163054 |   PyMPIComm_Type.tp_print = 0;",
    "278    163046 |   PyMPIInfo_Type.tp_print = 0;\n     279           |                 ^\n  >> 280    src/mpi4py.MPI.c:163050:23: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     281    163050 |   PyMPIErrhandler_Type.tp_print = 0;\n     282           |                       ^\n>> 283    src/mpi4py.MPI.c:163054:17: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     284    163054 |   PyMPIComm_Type.tp_print = 0;\n     285           |                 ^\n  >> 286    src/mpi4py.MPI.c:163059:22: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     287    163059 |   PyMPIIntracomm_Type.tp_print = 0;",
    "281    163050 |   PyMPIErrhandler_Type.tp_print = 0;\n     282           |                       ^\n  >> 283    src/mpi4py.MPI.c:163054:17: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     284    163054 |   PyMPIComm_Type.tp_print = 0;\n     285           |                 ^\n>> 286    src/mpi4py.MPI.c:163059:22: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     287    163059 |   PyMPIIntracomm_Type.tp_print = 0;\n     288           |                      ^\n  >> 289    src/mpi4py.MPI.c:163064:21: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     290    163064 |   PyMPITopocomm_Type.tp_print = 0;",
    "284    163054 |   PyMPIComm_Type.tp_print = 0;\n     285           |                 ^\n  >> 286    src/mpi4py.MPI.c:163059:22: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     287    163059 |   PyMPIIntracomm_Type.tp_print = 0;\n     288           |                      ^\n>> 289    src/mpi4py.MPI.c:163064:21: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     290    163064 |   PyMPITopocomm_Type.tp_print = 0;\n     291           |                     ^\n  >> 292    src/mpi4py.MPI.c:163069:21: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     293    163069 |   PyMPICartcomm_Type.tp_print = 0;",
    "287    163059 |   PyMPIIntracomm_Type.tp_print = 0;\n     288           |                      ^\n  >> 289    src/mpi4py.MPI.c:163064:21: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     290    163064 |   PyMPITopocomm_Type.tp_print = 0;\n     291           |                     ^\n>> 292    src/mpi4py.MPI.c:163069:21: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     293    163069 |   PyMPICartcomm_Type.tp_print = 0;\n     294           |                     ^\n  >> 295    src/mpi4py.MPI.c:163074:22: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     296    163074 |   PyMPIGraphcomm_Type.tp_print = 0;",
    "290    163064 |   PyMPITopocomm_Type.tp_print = 0;\n     291           |                     ^\n  >> 292    src/mpi4py.MPI.c:163069:21: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     293    163069 |   PyMPICartcomm_Type.tp_print = 0;\n     294           |                     ^\n>> 295    src/mpi4py.MPI.c:163074:22: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     296    163074 |   PyMPIGraphcomm_Type.tp_print = 0;\n     297           |                      ^\n  >> 298    src/mpi4py.MPI.c:163079:26: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     299    163079 |   PyMPIDistgraphcomm_Type.tp_print = 0;",
    "293    163069 |   PyMPICartcomm_Type.tp_print = 0;\n     294           |                     ^\n  >> 295    src/mpi4py.MPI.c:163074:22: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     296    163074 |   PyMPIGraphcomm_Type.tp_print = 0;\n     297           |                      ^\n>> 298    src/mpi4py.MPI.c:163079:26: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     299    163079 |   PyMPIDistgraphcomm_Type.tp_print = 0;\n     300           |                          ^\n  >> 301    src/mpi4py.MPI.c:163084:22: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     302    163084 |   PyMPIIntercomm_Type.tp_print = 0;",
    "296    163074 |   PyMPIGraphcomm_Type.tp_print = 0;\n     297           |                      ^\n  >> 298    src/mpi4py.MPI.c:163079:26: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     299    163079 |   PyMPIDistgraphcomm_Type.tp_print = 0;\n     300           |                          ^\n>> 301    src/mpi4py.MPI.c:163084:22: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     302    163084 |   PyMPIIntercomm_Type.tp_print = 0;\n     303           |                      ^\n  >> 304    src/mpi4py.MPI.c:163088:16: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     305    163088 |   PyMPIWin_Type.tp_print = 0;",
    "299    163079 |   PyMPIDistgraphcomm_Type.tp_print = 0;\n     300           |                          ^\n  >> 301    src/mpi4py.MPI.c:163084:22: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     302    163084 |   PyMPIIntercomm_Type.tp_print = 0;\n     303           |                      ^\n>> 304    src/mpi4py.MPI.c:163088:16: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     305    163088 |   PyMPIWin_Type.tp_print = 0;\n     306           |                ^\n  >> 307    src/mpi4py.MPI.c:163092:17: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     308    163092 |   PyMPIFile_Type.tp_print = 0;",
    "302    163084 |   PyMPIIntercomm_Type.tp_print = 0;\n     303           |                      ^\n  >> 304    src/mpi4py.MPI.c:163088:16: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     305    163088 |   PyMPIWin_Type.tp_print = 0;\n     306           |                ^\n>> 307    src/mpi4py.MPI.c:163092:17: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     308    163092 |   PyMPIFile_Type.tp_print = 0;\n     309           |                 ^\n  >> 310    src/mpi4py.MPI.c:163096:33: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     311    163096 |   __pyx_type_6mpi4py_3MPI_memory.tp_print = 0;",
    "305    163088 |   PyMPIWin_Type.tp_print = 0;\n     306           |                ^\n  >> 307    src/mpi4py.MPI.c:163092:17: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     308    163092 |   PyMPIFile_Type.tp_print = 0;\n     309           |                 ^\n>> 310    src/mpi4py.MPI.c:163096:33: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     311    163096 |   __pyx_type_6mpi4py_3MPI_memory.tp_print = 0;\n     312           |                                 ^\n  >> 313    src/mpi4py.MPI.c:163100:33: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     314    163100 |   __pyx_type_6mpi4py_3MPI__p_mem.tp_print = 0;",
    "308    163092 |   PyMPIFile_Type.tp_print = 0;\n     309           |                 ^\n  >> 310    src/mpi4py.MPI.c:163096:33: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     311    163096 |   __pyx_type_6mpi4py_3MPI_memory.tp_print = 0;\n     312           |                                 ^\n>> 313    src/mpi4py.MPI.c:163100:33: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     314    163100 |   __pyx_type_6mpi4py_3MPI__p_mem.tp_print = 0;\n     315           |                                 ^\n  >> 316    src/mpi4py.MPI.c:163107:34: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     317    163107 |   __pyx_type_6mpi4py_3MPI__p_greq.tp_print = 0;",
    "311    163096 |   __pyx_type_6mpi4py_3MPI_memory.tp_print = 0;\n     312           |                                 ^\n  >> 313    src/mpi4py.MPI.c:163100:33: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     314    163100 |   __pyx_type_6mpi4py_3MPI__p_mem.tp_print = 0;\n     315           |                                 ^\n>> 316    src/mpi4py.MPI.c:163107:34: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     317    163107 |   __pyx_type_6mpi4py_3MPI__p_greq.tp_print = 0;\n     318           |                                  ^\n  >> 319    src/mpi4py.MPI.c:163115:37: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     320    163115 |   __pyx_type_6mpi4py_3MPI__p_datarep.tp_print = 0;",
    "314    163100 |   __pyx_type_6mpi4py_3MPI__p_mem.tp_print = 0;\n     315           |                                 ^\n  >> 316    src/mpi4py.MPI.c:163107:34: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     317    163107 |   __pyx_type_6mpi4py_3MPI__p_greq.tp_print = 0;\n     318           |                                  ^\n>> 319    src/mpi4py.MPI.c:163115:37: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     320    163115 |   __pyx_type_6mpi4py_3MPI__p_datarep.tp_print = 0;\n     321           |                                     ^\n  >> 322    src/mpi4py.MPI.c:163119:36: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     323    163119 |   __pyx_type_6mpi4py_3MPI__p_keyval.tp_print = 0;",
    "317    163107 |   __pyx_type_6mpi4py_3MPI__p_greq.tp_print = 0;\n     318           |                                  ^\n  >> 319    src/mpi4py.MPI.c:163115:37: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     320    163115 |   __pyx_type_6mpi4py_3MPI__p_datarep.tp_print = 0;\n     321           |                                     ^\n>> 322    src/mpi4py.MPI.c:163119:36: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     323    163119 |   __pyx_type_6mpi4py_3MPI__p_keyval.tp_print = 0;\n     324           |                                    ^\n  >> 325    src/mpi4py.MPI.c:163122:37: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     326    163122 |   __pyx_type_6mpi4py_3MPI__p_message.tp_print = 0;",
    "320    163115 |   __pyx_type_6mpi4py_3MPI__p_datarep.tp_print = 0;\n     321           |                                     ^\n  >> 322    src/mpi4py.MPI.c:163119:36: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     323    163119 |   __pyx_type_6mpi4py_3MPI__p_keyval.tp_print = 0;\n     324           |                                    ^\n>> 325    src/mpi4py.MPI.c:163122:37: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     326    163122 |   __pyx_type_6mpi4py_3MPI__p_message.tp_print = 0;\n     327           |                                     ^\n  >> 328    src/mpi4py.MPI.c:163128:37: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     329    163128 |   __pyx_type_6mpi4py_3MPI__p_msg_p2p.tp_print = 0;",
    "323    163119 |   __pyx_type_6mpi4py_3MPI__p_keyval.tp_print = 0;\n     324           |                                    ^\n  >> 325    src/mpi4py.MPI.c:163122:37: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     326    163122 |   __pyx_type_6mpi4py_3MPI__p_message.tp_print = 0;\n     327           |                                     ^\n>> 328    src/mpi4py.MPI.c:163128:37: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     329    163128 |   __pyx_type_6mpi4py_3MPI__p_msg_p2p.tp_print = 0;\n     330           |                                     ^\n  >> 331    src/mpi4py.MPI.c:163150:37: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     332    163150 |   __pyx_type_6mpi4py_3MPI__p_msg_cco.tp_print = 0;",
    "326    163122 |   __pyx_type_6mpi4py_3MPI__p_message.tp_print = 0;\n     327           |                                     ^\n  >> 328    src/mpi4py.MPI.c:163128:37: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     329    163128 |   __pyx_type_6mpi4py_3MPI__p_msg_p2p.tp_print = 0;\n     330           |                                     ^\n>> 331    src/mpi4py.MPI.c:163150:37: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     332    163150 |   __pyx_type_6mpi4py_3MPI__p_msg_cco.tp_print = 0;\n     333           |                                     ^\n  >> 334    src/mpi4py.MPI.c:163157:38: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     335    163157 |   __pyx_type_6mpi4py_3MPI__p_msg_ccow.tp_print = 0;",
    "329    163128 |   __pyx_type_6mpi4py_3MPI__p_msg_p2p.tp_print = 0;\n     330           |                                     ^\n  >> 331    src/mpi4py.MPI.c:163150:37: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     332    163150 |   __pyx_type_6mpi4py_3MPI__p_msg_cco.tp_print = 0;\n     333           |                                     ^\n>> 334    src/mpi4py.MPI.c:163157:38: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     335    163157 |   __pyx_type_6mpi4py_3MPI__p_msg_ccow.tp_print = 0;\n     336           |                                      ^\n  >> 337    src/mpi4py.MPI.c:163172:37: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     338    163172 |   __pyx_type_6mpi4py_3MPI__p_msg_rma.tp_print = 0;",
    "332    163150 |   __pyx_type_6mpi4py_3MPI__p_msg_cco.tp_print = 0;\n     333           |                                     ^\n  >> 334    src/mpi4py.MPI.c:163157:38: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     335    163157 |   __pyx_type_6mpi4py_3MPI__p_msg_ccow.tp_print = 0;\n     336           |                                      ^\n>> 337    src/mpi4py.MPI.c:163172:37: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     338    163172 |   __pyx_type_6mpi4py_3MPI__p_msg_rma.tp_print = 0;\n     339           |                                     ^\n  >> 340    src/mpi4py.MPI.c:163179:36: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     341    163179 |   __pyx_type_6mpi4py_3MPI__p_msg_io.tp_print = 0;",
    "335    163157 |   __pyx_type_6mpi4py_3MPI__p_msg_ccow.tp_print = 0;\n     336           |                                      ^\n  >> 337    src/mpi4py.MPI.c:163172:37: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     338    163172 |   __pyx_type_6mpi4py_3MPI__p_msg_rma.tp_print = 0;\n     339           |                                     ^\n>> 340    src/mpi4py.MPI.c:163179:36: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     341    163179 |   __pyx_type_6mpi4py_3MPI__p_msg_io.tp_print = 0;\n     342           |                                    ^\n  >> 343    src/mpi4py.MPI.c:163192:33: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     344    163192 |   __pyx_type_6mpi4py_3MPI_Pickle.tp_print = 0;",
    "338    163172 |   __pyx_type_6mpi4py_3MPI__p_msg_rma.tp_print = 0;\n     339           |                                     ^\n  >> 340    src/mpi4py.MPI.c:163179:36: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     341    163179 |   __pyx_type_6mpi4py_3MPI__p_msg_io.tp_print = 0;\n     342           |                                    ^\n>> 343    src/mpi4py.MPI.c:163192:33: error: 'PyTypeObject' {aka 'struct _typ\n            eobject'} has no member named 'tp_print'\n     344    163192 |   __pyx_type_6mpi4py_3MPI_Pickle.tp_print = 0;\n     345           |                                 ^\n     346    In file included from src/MPI.c:4:\n     347    src/mpi4py.MPI.c: In function '__Pyx_ParseOptionalKeywords':\n     348    src/mpi4py.MPI.c:175146:21: warning: '_PyUnicode_get_wstr_length' i"
]